### PR TITLE
Run with Postgres database (BW-863)

### PR DESCRIPTION
**Pull request by cahrens**
Cromwell running with Postgres as deployed from Leonardo:

image

image

Demonstration that "replace-me" is coming through as the db password from Leo, when I (temporarily) displayed the password as an annotation.

image

### CODE REVIEW

File: cromwell-helm/templates/_helpers.tpl [hunk 1]
```diff
@@ -49,3 +49,17 @@ Selector labels
...
+
+{{/*
+Return postgres database user password.
+Lookup the existing secret values if they exist, or generate a random value
+*/}}
+{{- define "dbPassword" -}}
+{{- if .Values.db.password }}
+    {{- .Values.db.password -}}
+{{- else }}
+    {{- $randomValue := (randAlphaNum 32) -}}
```
Comment (cahrens) [inline cromwell-helm/templates/_helpers.tpl:61]: Stole this from Galaxy helm charts.

File: cromwell-helm/templates/postgres.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+###########################
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "app.fullname" . }}-postgres-depl
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      component: postgres
```
Comment (cahrens) [inline cromwell-helm/templates/postgres.yaml:12]: I don't know if I should make these labels more complicated (like we do for Cromwell, following Galaxy examples). TBH, I don't completely understand the point of having multiple labels.

Comment (kpierre13) [reply cromwell-helm/templates/postgres.yaml:12]: What are these labels used for? Do they show up somewhere or are they just for identification purposes?

Comment (cahrens) [reply cromwell-helm/templates/postgres.yaml:12]: TBH, I'm still pretty confused about how labels are used. But here is the documentation-- https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/

**Review by kpierre13 — APPROVED** [review]

**Review by salonishah11 — APPROVED** [review]
LGTM 👍

**Pull request merged**